src/fu-tool.c

```diff
@@ -878,10 +878,23 @@ fu_util_install (FuUtilPrivate *priv, gchar **values, GError **error)
 
 			/* is this component valid for the device */
 			task = fu_install_task_new (device, component);
+			if (!fu_engine_check_requirements (priv->engine,
+							   task, priv->flags | FWUPD_INSTALL_FLAG_IGNORE_VERFMT,
```

Comment:
would FORCE work here as well?

Reply:
Yes it does, adjusted to that much better thanks.